romana_setup.md

```diff
@@ -0,0 +1,117 @@
+# The `romana-setup` tool
+
+The `romana-setup` tool is used to provision and install Romana using the selected installation environment, linux distribution and stack type.
```

Comment:
Maybe add a comment here that this script is not meant for production deployments, which always require more specific setups, but instead is meant for evaluation and testing, etc...